app/src/main/java/com/hedvig/app/feature/insurance/ui/detail/yourinfo/YourInfoModel.kt

```diff
@@ -1,13 +1,14 @@
 package com.hedvig.app.feature.insurance.ui.detail.yourinfo
 
 sealed class YourInfoModel {
-    sealed class Header : YourInfoModel() {
-        object Details : Header()
-        object Coinsured : Header()
-    }
+    data class Header(val text: String) : YourInfoModel()
 
     data class Row(
         val label: String,
         val content: String
     ) : YourInfoModel()
+
+    data class Paragraph(val text: String) : YourInfoModel()
+
+    object Button : YourInfoModel()
```

Comment:
I think this name might be a bit too generic. It's not any old `Button`, this button always has a specific text and action. Perhaps `OpenChat`? Or something like that.